docs/user_guide/01_Reading_data.md

```diff
@@ -140,6 +139,29 @@ shp = utilities.read_file(input="/path/to/boxes_shapefile.shp")
 shp.head()
 ```
 
+If your shapefile does not include an `image_path` column, you must provide the raster path via `rgb_path`:
+
+```python
+from deepforest import utilities
+
+shp = utilities.read_file(
+    input="/path/to/boxes_shapefile.shp",
+    rgb_path="/path/to/OSBS_029.tif"
+)
+```
+
+If your shapefile also lacks a `label` column, you can assign one for all rows:
+
+```python
+from deepforest import utilities
+
+shp = utilities.read_file(
+    input="/path/to/boxes_shapefile.shp",
+    rgb_path="/path/to/OSBS_029.tif",
+    label="Tree"
+)
```

Comment:
We decided to use the existing `image_path` argument not create a new `reg_path`